bio-submission.txt

```diff
@@ -0,0 +1,41 @@
+Bio submission overview
+=======================
+
+Since btrfs has support for various raid levels it's submission path is more
+involved than a traditional filesystem. For the purposes of this document a
+raid1-based filesystem is going to be considered.
```

Comment:
IMHO it will be a good idea to be a bit more explicit. In my opinion this helps to frame the reader's expectations and it will be easier to understand the more detailed explanation following :

"In this scenario for every write request initiated by the user the filesystem will have the responsibility of mapping it to 2 devices underlying devices"